src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php

```diff
@@ -27,24 +28,52 @@
 class ConstraintViolationBuilder implements ConstraintViolationBuilderInterface
 {
     private string $propertyPath;
+    private ?string $message = null;
```

Comment:
no need to make this nullable:
```suggestion
    private string $translatedMessage;
```

Reply:
so the point here is that it will be eventually initialized?

right now this is only initialized as `$this->message ??= $this->translateMessage()`

any access to this property prior to that place will throw an exception